docs/pages/messages/unnecessary-package-task-syntax.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Unnecessary Package Task Syntax
```

Comment:
Sentence casing.

Reply:
Looking at the Next errors, these titles are actual sentences, versus the fragments that we have here. I'd learn towards title capitalization if we're keeping the fragments, otherwise we should make these full sentences

Reply:
It's more of a general Vercel style guide thing like here: https://vercel.com/blog/introducing-conformance#all-while-elevating-application-security

Reply:
(The Next.js errors are actually wrong, according to the style guide.)